src/lib/cssCustomProperties.js

```diff
@@ -0,0 +1,6 @@
+// This is needed to generate CSS with definitions to be used in docs directly (not in shadow DOM)
```

Comment:
Comment says "to be used in docs directly". If it is true, move it to `docs` folder. If it is used generally to generate styles (despite the docs), improve the comment.

Reply:
This file is used to generate a CSS file that only contains CSS custom property definitions.

It can be used anywhere. Its just that outside of the context of docs I'm having hard time coming up with valid use case.

I dislike moving it to the `docs` folder because:
* It is very similar to `index.js` and serves exactly the same purpose, I think they should remain together
* In `docs` folder there is no file that imports from `lib`, it is one of the two entry points how the lib can be build

We could move it into a subfolder named `entryPoints`, but:
* that sort of leaves the `*.scss` files hanging there
* having a `src/index.js` just somehow makes sense to me.

So I think the best thing to do would be to change the comment to:

> // This is needed to generate CSS file containing only CSS custom property definitions
// This file is needed to make custom property values accessible in the documentation


@bedrich-schindler Would that work for you?

Reply:
OK